Modules/Relationships/src/Relationships.Domain/DomainErrors.cs

```diff
@@ -35,10 +35,10 @@ public static DomainError RelationshipIsNotInCorrectStatus(RelationshipStatus ex
             $"The relationship has to be in status '{expectedStatus}' to perform this action.");
     }
 
-    public static DomainError RelationshipIsInIncorrectStatus(RelationshipStatus expectedStatus)
+    public static DomainError RelationshipIsInIncorrectStatus(RelationshipStatus undesiredStatus)
     {
         return new DomainError("error.platform.validation.relationshipRequest.relationshipIsInIncorrectStatus",
-            $"The relationship cannot to be in status '{expectedStatus}' to perform this action.");
+            $"The relationship cannot to be in status '{undesiredStatus}' to perform this action.");
     }
```

Comment:
As described in a comment in my previous review, I would change the error code to something like "*HasToBeInStatus" (or however we call it in other places), and in the error message list all statuses in which the operation is possible.